README.md

```diff
@@ -71,6 +71,7 @@ Most controls are settable in the configuration file, see the section below.
 
 Paths
 ------
+```
```

Comment:
Use a list like the following rather than \`\`\`

* `$bin` - Compiled binaries